calm/draft/2024-04/meta/core.json

```diff
@@ -232,6 +241,12 @@
           }
         }
       }
+    },
+    "metadata": {
```

Comment:
Do you have an example of how you plan to use this, trying to understand the decision to use an array of object rather than say a key/value string pair.

Reply:
Spoke offline